Add procedures for incremental exports (#2071)
theforeman/foreman-documentation: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc

[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
= Exporting a Content View Version in a Syncable Format

> You can export a version of a Content View to a syncable format that you can use to create your custom CDN. Once exported, do either of the following:

Reviewer: ```suggestion
You can export a version of a Content View to a syncable format that you can use to create your custom CDN.
After you have exported the Content View, you can do either of the following:
```

> * Synchronize the content from the custom CDN over HTTP/HTTPS.

Reviewer: ```suggestion
* Synchronize the content from your custom CDN over HTTP/HTTPS.
```

> * Import the content via `hammer content-import` which is available if both the Export and Import servers are {ProductVersion} and above.

Reviewer: ```suggestion
* Import the content using `hammer content-import`.
Note that this requires both the Export and Import servers are {ProductVersion} and above.
```

I am unsure about the second sentence because the docs are specifically built for {ProductVersion} anyway. Maybe reword so that users ensure both the export and import {ProjectServer} are on the same/target version?

[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
= Exporting a Content View Version in a Syncable Format

You can export a version of a Content View to a syncable format that you can use to create your custom CDN.
After you have exported the Content View, you can do either of the following:

* Synchronize the content from your custom CDN over HTTP/HTTPS.
* Import the content using `hammer content-import`.

> Note that this requires both the Export and Import servers are the same {ProductVersion}.

Reviewer: ```suggestion
Note that this requires both the Export and Import servers to run {Project} {ProductVersion}.
```

> Note that this requires both the Export and Import servers are the same {ProductVersion} and above.

Reviewer: This sounds off to me. Please reword.

I am also unsure if "current version or above" make sense in different branches.
